indico/modules/categories/templates/display/sidebar.html

```diff
@@ -1,89 +1,179 @@
-{% from 'attachments/_attachments.html' import render_attachments as _render_attachments %}
+{% macro _render_attachment_file(attachment) %}
+    {% set previewable = (attachment.file.is_previewable and not g.static_site) or false %}
+    {% set visible_link = attachment.type.name == 'link' and (not attachment.is_protected or attachment.can_access(session.user)) %}
+    {# TODO: Add preview feature #}
+    <li>
+        <ind-attachment>
+            <a data-content-type="{{ attachment.file.content_type }}"
+               class=""
+               href="{{ attachment.link_url if visible_link else attachment.download_url }}"
+               target="blank"
```

Comment:
```suggestion
               target="_blank"
```

"blank" works the first time, but if you click another link it would replace the previously opened one (since "blank" is just the name of the new tab/window, only `_blank` has a special meaning)

Reply:
Typo! Thanks